src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Data/Given_BindingExpression.cs

```diff
@@ -0,0 +1,585 @@
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Private.Infrastructure;
+using Uno.Disposables;
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Data;
+
+[TestClass]
+public class Given_BindingExpression
```

Comment:
Can you validate that those new tests pass on the winui version of the samples app?

Reply:
I wrote them on WinUI first then made our implementation pass, so they should definitely pass.